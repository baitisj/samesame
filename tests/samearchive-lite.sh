DIFF=diff
SED=sed

WORK=${srcdir}/work
TEST1=${WORK}/test1
TEST2=${WORK}/test2
LIST1=${WORK}/test1.lst
OUTPUT=${WORK}/test.out
CHECK=${srcdir}/tests/samearchive-lite.out
CMD=${srcdir}/samearchive-lite

${CMD} ${TEST1} ${TEST2} < ${LIST1} | ${SED} -e 's/${srcdir}/./g' > ${OUTPUT}
${DIFF} -u ${CHECK} ${OUTPUT}
