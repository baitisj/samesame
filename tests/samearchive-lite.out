11	./work/test1/file1a	./work/test2/file1a	=	2	1
11	./work/test1/file3a	./work/test2/file3a	=	1	1
11	./work/test1/file2a	./work/test2/file2a	=	3	1
11	./work/test1/file2c	./work/test2/file2c	=	3	1
11	./work/test1/file2b	./work/test2/file2b	=	3	1
11	./work/test1/file1b	./work/test2/file1b	=	2	1
